cl/audio/models.py

```diff
@@ -89,15 +90,15 @@ class Audio(AbstractDateTimeModel):
         help_text="The location, relative to MEDIA_ROOT, on the CourtListener "
         "server, where encoded file is stored",
         upload_to=make_upload_path,
-        storage=IncrementingFileSystemStorage(),
+        storage=AWSMediaStorage(),
         blank=True,
         db_index=True,
     )
     local_path_original_file = models.FileField(
         help_text="The location, relative to MEDIA_ROOT, on the CourtListener "
         "server, where the original file is stored",
         upload_to=make_upload_path,
-        storage=IncrementingFileSystemStorage(),
+        storage=AWSMediaStorage(),
         db_index=True,
```

Comment:
Seems, either the help text should be updated, or the naming convention should change?  

In LASC we used the naming convention `filepath_s3`.  I suppose you could keep the naming convention and update the help text to reflect its not on the CL server.